profiles/base.yaml

```diff
@@ -129,6 +129,7 @@ ambush_location: back
 warhorn:
 prayer_mat:
 paladin_use_badge: false
+offhand_thrown:
```

Comment:
What do you think of making this `offhand_thrown: false` instead?

Reply:
Makes sense to me